### PR TITLE
feat(std/http): Add Cookie value validation 

### PR DESCRIPTION
**Pull request by getspooky**

### CODE REVIEW

**Comment by bartlomieju** [comment]
@nayeemrmn can you take a look?

File: std/http/cookie.ts [hunk 1]
```diff
@@ -114,6 +115,31 @@ function validatePath(path: string | null): void {
...
+function validateCookieValue(value: string | null): void {
+  if (value == null) return;
+  for (let i = 0; i < value.length; i++) {
+    const c = value.charAt(i);
+    if (
+      c < String.fromCharCode(0x21) || c == String.fromCharCode(0x22) ||
+      c == String.fromCharCode(0x2c) || c == String.fromCharCode(0x3b) ||
+      c == String.fromCharCode(0x5c) || c == String.fromCharCode(0x7f)
+    ) {
+      throw new Error("RFC2616 cookie value cannot have '" + c + "'");
```
Comment (bartlomieju) [inline std/http/cookie.ts:132]: Nitpick, include cookie name in the error

File: std/http/cookie.ts [hunk 2]
```diff
@@ -114,6 +115,31 @@ function validatePath(path: string | null): void {
...
+    }
+    if (c > String.fromCharCode(0x80)) {
+      throw new Error(
+        "RFC2616 cookie value can only have US-ASCII chars" +
+          c.charCodeAt(0).toString(16),
+      );
+    }
```
Comment (bartlomieju) [inline std/http/cookie.ts:139]: Ditto

**Review by bartlomieju — CHANGES_REQUESTED** [review]

File: std/http/cookie.ts (hunk 1 above)
Comment (getspooky) [reply std/http/cookie.ts:132]: what do you think ? 
 ```js
"RFC2616 cookie '" + name + "' cannot have '" + c + "' as value" 
```

File: std/http/cookie.ts (hunk 2 above)
Comment (getspooky) [reply std/http/cookie.ts:139]:
```js
 "RFC2616 cookie '" + name + "' can only have US-ASCII chars as value" +
          c.charCodeAt(0).toString(16)
```

**Comment by getspooky** [comment]
@bartlomieju i made changes

**Review by bartlomieju — APPROVED** [review]
LGTM, thanks @getspooky

**Pull request merged**